fs/filter/filter.go

```diff
@@ -330,23 +330,30 @@ func (f *Filter) DirContainsExcludeFile(ctx context.Context, fremote fs.Fs, remo
 }
 
 // Include returns whether this object should be included into the
-// sync or not
+// sync or not and logs the reason for exclusion if not included
 func (f *Filter) Include(remote string, size int64, modTime time.Time, metadata fs.Metadata) bool {
 	// filesFrom takes precedence
 	if f.files != nil {
 		_, include := f.files[remote]
+		if !include {
+			fs.Debugf(nil, "Excluded (FilesFrom Filter): %s", remote)
```

Comment:
These debugs should be

    fs.Debugf(remote, "Excluded (FilesFrom Filter)", remote)

Then they will appear correct in the logs.